improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -992,28 +1072,26 @@ def _location_and_scale_parameters_to_probabilities(
         distribution = self.distribution(
             *self.shape_parameters,
             loc=location_parameter.data.flatten(),
-            scale=np.sqrt(scale_parameter.data.flatten()))
+            scale=np.sqrt(scale_parameter.data.flatten()),
+        )
 
         probability_method = distribution.cdf
-        if relative_to_threshold == 'above':
+        if relative_to_threshold == "above":
             probability_method = distribution.sf
 
         for index, threshold in enumerate(thresholds):
             probabilities[index, ...] = np.reshape(
-                probability_method(threshold),
-                probabilities.shape[1:]) \
-                # pylint: disable=unsubscriptable-object
+                probability_method(threshold), probabilities.shape[1:]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Value 'probabilities.shape' is unsubscriptable](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5354756)